deploy-manage/security/fips-es.md

```diff
@@ -0,0 +1,167 @@
+---
+applies_to:
+  deployment:
+    self: ga
+products:
+  - id: elasticsearch
+---
+
+# FIPS compliance for {{es}} [fips-elasticsearch]
+
+
+::::{important}
+The JVM bundled with {{es}} is not configured for FIPS 140-2. You must configure an external JDK with a FIPS 140-2 certified Java Security Provider. Refer to the {{es}} [JVM support matrix](https://www.elastic.co/support/matrix#matrix_jvm) for supported JVM configurations. See [subscriptions](https://www.elastic.co/subscriptions) for required licensing.
+::::
+
+
+Compliance with FIPS 140-2 requires using only FIPS approved / NIST recommended cryptographic algorithms. Generally this can be done by the following:
+
+* Installation and configuration of a FIPS certified Java security provider.
+* Ensuring the configuration of {{es}} is FIPS 140-2 compliant as documented below.
+* Setting `xpack.security.fips_mode.enabled` to `true` in [`elasticsearch.yml`](/deploy-manage/stack-settings.md). Note - this setting alone is not sufficient to be compliant with FIPS 140-2.
+
+
+:::{note}
+If you are running {{es}} through {{eck}}, refer to [ECK FIPS compatibility](/deploy-manage/deploy/cloud-on-k8s/deploy-fips-compatible-version-of-eck.md).
+:::
+
+## Configuring {{es}} for FIPS 140-2 [_configuring_es_for_fips_140_2]
+
+Detailed instructions for the configuration required for FIPS 140-2 compliance is beyond the scope of this document.  It is the responsibility of the user to ensure compliance with FIPS 140-2. {{es}} has been tested with a specific configuration described below. However, there are other configurations possible to achieve compliance.
+
+Here is a high-level overview of the required configuration:
+
+* Use an externally installed Java installation. The JVM bundled with {{es}} is not configured for FIPS 140-2.
+* Install a FIPS certified security provider .jar file(s) in {{es}}'s `lib` directory.
+* Configure Java to use a FIPS certified security provider ([see below](#java-security-provider)).
+* Configure {{es}}'s security manager to allow use of the FIPS certified provider ([see below](#java-security-manager)).
+* Ensure the keystore and truststore are configured correctly ([see below](#keystore-fips-password)).
+* Ensure the TLS settings are configured correctly ([see below](#fips-tls)).
+* Ensure the password hashing settings are configured correctly ([see below](#fips-stored-password-hashing)).
+* Ensure the cached password hashing settings are configured correctly ([see below](#fips-cached-password-hashing)).
+* Configure `elasticsearch.yml` to use FIPS 140-2 mode, see ([below](#configuring-es-yml)).
+* Verify the security provider is installed and configured correctly ([see below](#verify-security-provider)).
+* Review the upgrade considerations ([see below](#fips-upgrade-considerations)) and limitations ([see below](#fips-limitations)).
```

Comment:
Error on line 44: `fips-limitations` does not exist in deploy-manage/security/fips-es.md.
But checkout line 160: # Limitations [fips-limitations]

Also weird... The second link in Line 44 links correctly to Limitations content both locally and in PR preview. 
It's the very last link in https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2136/deploy-manage/security/fips-es#_configuring_es_for_fips_140_2.

@colleenmcginnis I'm missing something silly, but I don't know what it is.